snapcraft.yaml

```diff
@@ -1633,6 +1647,7 @@ parts:
 
       OLDPATH=$PATH
       python3 -m pip install pyelftools craft-parts==1.14.0
+      git config --global --add safe.directory /data
```

Comment:
The point is... what is this "data" folder?

Reply:
Okay, wait.. let me remove the github action files and try to build this locally

Reply:
Yes, tested locally.. no proble, you can give that a try too